msteams-platform/m365-apps/teamsjs-support-m365.md

```diff
@@ -229,6 +229,24 @@ The following table lists host application support for TeamsJS capabilities that
             <td></td>
             <td></td>
         </tr>
+        <tr>
+            <th>search <img src="./images/preview-badge.png" alt="Badge indicating this capability is in preview" /></th>
+            <td></td>
+            <td></td>
+            <td>&#x2713;*</td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td></td>
+            <td>&#x2713;*</td>
```

Comment:
Oops, wrong Outlook column (this is for "new Outlook", with the PRE on the logo); should be marked for classic win32 Outlook in the column right before. (Same thing for table in "search" section of doc)